docs/_documentations/che-installinfo.md

```diff
@@ -31,15 +32,16 @@ Install Eclipse Che with HTTPS so that Codewind functions properly. See the [Ins
 ### Installing Che
 The fastest way to install Eclipse Che for Codewind is to use the `chectl` CLI. To install the `chectl` CLI tool, see [Installing the chectl management tool](https://www.eclipse.org/che/docs/che-7/installing-the-chectl-management-tool/).
 
-After you install `chectl`, download the [codewind-checluster.yaml](https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.12.0/setup/install_che/che-operator/codewind-checluster.yaml) file.
+After you install `chectl`, download the [codewind-checluster.yaml](https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.13.0/setup/install_che/che-operator/codewind-checluster.yaml) file.
  - You can modify this file, but leave the `spec.server.cheWorkspaceClusterRole` field set to `eclipse-codewind` and the `spec.storage.preCreateSubPaths` field set to `true`.
+ - If you are installing Che with publicly signed certificates (such as on OpenShift on IBM Cloud), you must set `spec.server.selfSignedCert` to false.
```

Comment:
What about, "If you install Che with publicly signed certificates, such as on OpenShift on IBM Cloud, set the `spec.server.selfSignedCert` value to **false**."

Reply:
The original and the edited version sound very similar to me. Here's what I was trying to accomplish with the edited version:
- Remove the parenthesis. Unless they're part of code, we should generally have no parenthesis in our documentation.
- Simplify with "If you install" instead of "If you are installing" and "set" instead of "you must set."
Is there a way to accomplish this and still retain the original meaning?

Reply:
@sishida My bad, I completely misread the edited statement. That sounds good to me